asreview/state/__init__.py

```diff
@@ -13,10 +13,11 @@
 # limitations under the License.
 
 from asreview.state.base import BaseState
-from asreview.state.dict import DictState
+from asreview.state.legacy.dict import DictState
+from asreview.state.legacy.hdf5 import HDF5StateLegacy
 from asreview.state.hdf5 import HDF5State
```

Comment:
What would be a good name for the new model?

Reply:
Maybe SqlState? It's not entirely accurate, because there are also other files in there, but it fits in line with jsonstate and hdf5state.

Reply:
Let's do SqlStateV1.